### PR TITLE
fix link in introduction block being smaller than normal text

### PR DESCRIPTION
**Pull request by jonaspiterek**

### CODE REVIEW

**Comment by sneridagh** [comment]
@danalvrz could you please take a look?

File: src/theme/blocks/_introduction.scss [hunk 1]
```diff
@@ -19,6 +19,10 @@
     em span {
       font-style: italic;
     }
+    > a {
+      font-size: 24px;
```
Comment (tisto) [inline src/theme/blocks/_introduction.scss:23]: Just wondering. Can't we somehow incorporate the font-size and line-height from the standard text? Or even better get rid of a fixed setting of those values? Why are links styled differently than regular text in any case?

Comment (danalvrz) [reply src/theme/blocks/_introduction.scss:23]: @jonaspiterek I think moving the `> a` selector up one nesting level should work, so it uses the `introduction()` mixin that is there already. Have you tried this?

Comment (sneridagh) [reply src/theme/blocks/_introduction.scss:23]: @jonaspiterek status?

Comment (jonaspiterek) [reply src/theme/blocks/_introduction.scss]: Updated & tested. /cc @danalvrz

**Pull request merged**